### PR TITLE
Trimmomatic 0.33 (new formula)

### PR DESCRIPTION
**Pull request by tseemann**
Trimmomatic performs a variety of useful trimming tasks for illumina paired-end and single ended data.The selection of trimming steps and their associated parameters are supplied on the command line.

### CODE REVIEW

**Comment by tseemann** [comment]
I am unsure what is failing here. 

The trimmomatic help is being displayed, so it is running correctly. 

Is this a non-zero return code issue?

```
     Testing trimmomatic
==> /usr/local/Cellar/trimmomatic/0.33/bin/trimmomatic
Usage: 
       PE [-threads <threads>] [-phred33|-phred64] [-trimlog <trimLogFile>] [-basein <inputBase> | <inputFile1> <inputFile2>] [-baseout <outputBase> | <outputFile1P> <outputFile1U> <outputFile2P> <outputFile2U>] <trimmer1>...
   or: 
       SE [-threads <threads>] [-phred33|-phred64] [-trimlog <trimLogFile>] <inputFile> <outputFile> <trimmer1>...
Error: trimmomatic: failed
Failed executing: /usr/local/Cellar/trimmomatic/0.33/bin/trimmomatic 
/usr/local/Library/Homebrew/formula.rb:728:in `system'
/usr/local/Library/Homebrew/formula.rb:685:in `open'
/usr/local/Library/Homebrew/formula.rb:685:in `system'
/usr/local/Library/Taps/homebrew/homebrew-science/trimmomatic.rb:31:in `test'
/usr/local/Library/Homebrew/formula.rb:637:in `run_test'
/usr/local/Library/Homebrew/extend/fileutils.rb:17:in `mktemp'
/usr/local/Library/Homebrew/formula.rb:634:in `run_test'
/usr/local/Library/Homebrew/cmd/test.rb:38:in `test'
/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/timeout.rb:67:in `timeout'
/usr/local/Library/Homebrew/cmd/test.rb:37:in `test'
/usr/local/Library/Homebrew/cmd/test.rb:15:in `each'
/usr/local/Library/Homebrew/cmd/test.rb:15:in `test'
/usr/local/Library/brew.rb:135:in `send'
/usr/local/Library/brew.rb:135
```

**Comment by saketkc** [comment]
Does it print to `stderr`? One strategy is to pipe it to `grep`:
https://github.com/Homebrew/homebrew-science/blob/master/samtools.rb#L63

**Comment by tseemann** [comment]
I can't test it, as "brew test xxxx" is broken on Linuxbrew for me. I've filed an issue.

**Comment by sjackman** [comment]
> Is this a non-zero return code issue?

Yes, likely. You can assert the expected exit status (often 1) like so (just an example)
```ruby
assert_match "seqtk", shell_output("#{bin}/seqtk 2>&1", 1)
```

**Comment by tseemann** [comment]
It was returning ```$? eq 1```.  I've squashed the commits and fixed the test.

File: trimmomatic.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
```
Comment (sjackman) [inline trimmomatic.rb]: Please delete this blank line.

**Comment by sjackman** [comment]
@BrewTestBot test this please

**Pull request closed without merging**